### DISCUSSION TITLE
Inquiry on Deploying LibreChat with Azure OpenAI on Railway

### DISCUSSION THREAD

**Discussion by qw1311581** [danny-avila/LibreChat#2886] [Q&A] [answered]
I am currently working on deploying a project using LibreChat and its support for Azure OpenAI. However, I have encountered a technical challenge that I would like to seek your assistance with.
According to the documentation, deploying Azure OpenAI API requires the use of a librechat.yaml file. Unfortunately, it appears that Railway does not support Docker Compose, which raises a concern for me.
Could you please clarify whether this limitation means that it is not possible to utilize the Azure OpenAI API on Railway? Any guidance or alternative deployment strategies you could suggest would be greatly appreciated.

Thank you very much for your time and assistance.

**Comment by fuegovic** [comment, marked as answer]
You can utilize the librechat.yaml file with Railway by hosting it on a publicly accessible URL and including it in the variables, for example: `CONFIG_PATH=https://raw.githubusercontent.com/danny-avila/LibreChat/main/librechat.example.yaml`.

For serious deployment, it is advisable to host on a proper VPS. We offer a comprehensive guide for deploying on Digital Ocean, which can also serve as a foundation for deploying on any other VPS of your choice.

https://www.librechat.ai/docs/remote/digitalocean
